be/src/agent/cgroup_cpu_ctl.cpp

```diff
@@ -91,6 +91,9 @@ Status CgroupCpuCtl::write_cg_sys_file(std::string file_path, int value, std::st
         return Status::InternalError<false>("{} write sys file failed", msg);
```

Comment:
also need close fd before return here？

Reply:
good question.Refer  google result or some file desc open code in doris, no need close when open failed.

Reply:
yes